packages/react-native-codegen/android/build.gradle

```diff
@@ -42,5 +43,25 @@ task('buildCodegenCLI', type: Exec) {
     nodeModulesDir.mkdirs();
     outputs.dirs(libDir, nodeModulesDir)
 
-    commandLine("$codegenRoot/scripts/oss/build.sh")
+    if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+      // Convert path to Linux format: use canonical path to strip it off relative elements in the middle of the string.
+      // Then replace baskslashes with slashes, remove leading colon, add leading slash.
+      // Eg. D:\path1\sub2/.. -> /D/path1/path2
+      String canonicalPath = new File(codegenRoot).getCanonicalPath()
+      String linuxPath = canonicalPath.replace('\\', '/');
+      linuxPath = linuxPath.replace(':', '')
+      linuxPath = '/' + linuxPath
+      codegenRoot = linuxPath
+      
+      // Get the location of bash in the system; assume environment variable created to store it.
+      String bashHome = "$System.env.REACT_WINDOWS_BASH"
```

Comment:
Does Git Bash not add anything to path or environment vars already? Or does it install to the same location always which we could rely on instead?

Reply:
I don't remember which of the below options I went with:
![image](https://user-images.githubusercontent.com/12816515/101396155-c4e0c900-387f-11eb-9922-0831ed99f141.png)
but in my case Git Bash was not in my path, so I had to add it. From these descriptions, I think I went with the recommended, middle option, which we can assume most people will go with: git added to path, but not git-bash.
I think you can also adjust the installation location.